pyproject.toml

```diff
@@ -57,13 +34,27 @@ exclude = '.*/config_models/.*\.py$'
 exclude = [
     ".eggs",
     ".tox",
+    ".git",
+    ".mypy_cache",
+    ".venv",
+    ".pytest_cache",
+    ".ruff_cache",
     "build",
+    "_build",
+    "dist",
+    "buck-out", # <- is this something we still have?
     "compat.py",
     "__init__.py",
     "**/datadog_checks/dev/tooling/templates/*",
+    "**/datadog_checks/dev/tooling/signing.py",
     "**/datadog_checks/*/vendor/*",
+    # Avoid linting or formatting autogenerated files from ddev validate models
+    # They are formatted with black and ruff does not yet expose an api to format them
```

Comment:
```suggestion
    # They are formatted with Black, and Ruff does not yet offer an API to format them
```

Reply:
Updated this and the rest of the comments on the latest version, thanks @estherk15 !